### DISCUSSION TITLE
Mojo🔥 now available on Mac!

### DISCUSSION THREAD

**Discussion by jackos** [modular/modular#1075] [Announcements]
Hi Everyone! 

Today is the day! Mojo for Mac is live! 🔥😱🚀

[Download right now here](https://developer.modular.com/download)

[Read our launch blog post here](https://www.modular.com/blog/mojo-is-now-available-on-mac)

**Comment by lightcax** [comment]
When will it be supported for windows, I hope the modular can speed it up, thanks!

**Reply by jackos** [reply]
WSL and docker is supported on Windows, native support is being worked on now thanks for your patience 😀

**Comment by Moosems** [comment]
Is Mac Intel also around the corner?

**Reply by soraros** [reply]
It is tracked in #1092.

**Comment by vjpr** [comment]
First time user experience:

```
$ modular install mojo

modular: error: please run `modular auth` before attempting to install a package
```

Googled the error message and came here: https://docs.modular.com/mojo/roadmap.html#mojo-sdk-known-issues

```
$ modular auth xxx
```

---

```
$ modular install mojo

Exception: Could not find libpython. Check your Python installation and ensure that $LD_LIBRARY_PATH includes the location to the installed Python library (e.g. /usr/lib/x86_64-linux-gnu).
modular: error: failed to run python:
```

From https://docs.modular.com/mojo/roadmap.html#mojo-sdk-known-issues, now looking into: https://github.com/modularml/mojo/issues/551

---

Please make a CLI that handles these errors for me, and can determine what Python version manager I am using and suggest a fix.

**Comment by cjohnson318** [comment]


**Reply by iamtimdavis** [reply]
Please see #1078

**Reply by cjohnson318** [reply]
Thank you. I am a dunce.
